common-items.lic

```diff
@@ -80,13 +80,14 @@ module DRCI
     result =~ /You tap/
   end
 
-  def refill_lockpick_container(lockpick_type, hometown, container)
+  def refill_lockpick_container(lockpick_type, hometown, container, cost = 125)
```

Comment:
I'm not sure I like having a default value here... but if we do, I suggest erring on the side of the most expensive lockpick type.